microSALT/utils/scraper.py

```diff
@@ -438,6 +438,7 @@ def scrape_blast(self, type="", file_list=[]):
                         self.name, str(e)
                     )
                 )
+        return hypo
```

Comment:
This hypo variable is something extra

Reply:
Not following exactly, but this return was added here in order to be able to check the outcome of the scraping in tests. Optimally one would have refactored the code more, so as not to save to the database inside the scraping function at all like is done now, but this was the smallest change I could find that would do the trick.